The-SMIRNOFF-force-field-format.md

```diff
@@ -51,26 +75,85 @@ If not specified by a `*_unit` attribute, the [standard OpenMM unit system](http
+The file consists of multiple `Force` sections, with each section specifying the typing rules used to assign parameters for that potential term:
+```XML
+<BondForce potential="harmonic" length_unit="angstroms" k_unit="kilocalories_per_mole/angstrom**2">
+   <Bond smirks="[#6X4:1]-[#6X4:2]" length="1.526" k="620.0"/>
+   <Bond smirks="[#6X4:1]-[#1:2]" length="1.090" k="680.0"/>
+   ...
+</BondForce>
+<AngleForce potential="harmonic" angle_unit="degrees" k_unit="kilocalories_per_mole/radian**2">
+   <Angle smirks="[a,A:1]-[#6X4:2]-[a,A:3]" angle="109.50" k="100.0"/>
+   <Angle smirks="[#1:1]-[#6X4:2]-[#1:3]" angle="109.50" k="70.0"/>
+   ...
+</AngleForce>
+...
+```
 Each `Force` section specifies the functional form for the interaction using the `potential` attribute.
 Common defaults are defined, but the goal is to eventually allow these to be overridden by alternative choices or even algebraic expressions in the future.
 The format will distinguish between functional forms available in all common molecular simulation packages and those experimental features available in a few packages (especially OpenMM, which supports a flexible set of custom forces defined by algebraic expressions) with an *EXPERIMENTAL* category.
 
 Many of the specific forces are implemented as discussed in the [OpenMM Documentation](http://docs.openmm.org/latest/userguide/theory.html); see especially [Section 19 on Standard Forces](http://docs.openmm.org/latest/userguide/theory.html#standard-forces) for mathematical descriptions of these functional forms.
 In some cases, typically for consistency with the AMBER force field philosophy motivating some of the authors, we do some manipulation of parameters from these files as discussed below in "Parameter sections".
 
-**Charges**: In keeping with the AMBER force field philosophy, especially as implemented in small molecule force fields such as [GAFF](http://ambermd.org/antechamber/gaff.html), [GAFF2](https://mulan.swmed.edu/group/gaff.php), and [parm@Frosst](http://www.ccl.net/cca/data/parm_at_Frosst/), we at least initially treat partial charges as something to be obtained separately from the rest of the force field (BAIT, which includes bonds, angles, impropers, and torsions; and vdW terms), typically via QM calculations or a method such as the [AM1-BCC](https://dx.doi.org/10.1002/jcc.10128) approach from Christopher Bayly, thus, for system setup we provide the option of specifying a charging method, though charges are not normally specified in the FFXML itself.
-For other force fields with "library"-style charges, we could introduce a new section providing specific charges via SMIRKS pattern, though this is not yet implemented. Bond charge corrections, however, are supported as we discuss below.
+**QUESTION:** Since we're defining potentials instead of forces, do we want to remove the term `Force` from these tags and instead use the simpler `<Bonds/>`, `<Angles/>`, `<Torsions/>`, etc?
```

Comment:
That sounds good to me, and makes things like `BondChargeCorrections` sound less like they don't belong.

Reply:
What David says except BondChargeCorrections should perhaps become the more general BondChargeIncrements. I called them corrections for AM1-BCC because I knew I was adding them to correct AM1 charges already present; in the general case in offxml we are just adding them. Might we want an AtomCharge keyword for putting on library charges?  We could use this for smirks-based precharging for standard formal charge templates for e.g. carboxylates as with MMFF94.